ebmlite/encoding.py

```diff
@@ -218,7 +219,7 @@ def encodeBinary(val, length=None):
                          (len(val), length))
 
 
-def encodeString(val, length=None):
+def encodeString(val: ByteString, length: Optional[int] = None):
```

Comment:
Should also include `str`... from the EBML spec, the subset-of-ASCII limitation is on the storage, and the data is still considered to be strings.